templates/motor-insurance/inputs/selected-policy-option.js

```diff
@@ -0,0 +1,71 @@
+import { html, templates } from '/src/main.js';
+
+export default (name, key, output) => {
+    if (!output) {
+        return '';
+    }
+
+    return html`
+    <div class="field">
+        <span class="field__name">${name}</span>
+        <div class="field__inputs group group--merged">
+            <input
+                type="radio"
+                name="_${key}-options"
+                id="${key}-yes"
+                value=""
+                required
+                onclick="document.querySelector('#${key}-options-list').style.display = 'block';  document.querySelector('#${key}-option-null').checked = false;"
```

Comment:
This inlined function is a little gnarly. Is it possible to define a function outside of the template and call it within?

```javascript
function onclick() { // probably a better name
    document.querySelector(`#${key}-option-list`).style.display = 'block';
    document.querySelector(`#${key}-option-null`).checked = false;
}
```